doc/ref_arch/kubernetes/chapters/chapter03.rst

```diff
@@ -150,14 +151,26 @@ will work with kubelet). Examples of this type of runtime include containerd
 (with CRI plugin) and cri-o, which is built specifically to work with
 Kubernetes.
 
-To fulfil ``inf.vir.01`` the architecture should support a container runtime
-which provides the isolation of Operating System kernels.
+For Kubernetes, the important interface to consider for container management is the
+`Kubernetes Container Runtime Interface
+(CRI) <https://kubernetes.io/blog/2016/12/container-runtime-interface-cri-in-kubernetes/>`__.
+This is an interface specification for any container runtime to enable it to integrate
+with the kubelet on a Kubernetes Node. The CRI decouples the kubelet from the runtime
+that is running in the Host OS. This means that the code required to integrate the
```

Comment:
Host OS -- should we use "Node OS"

Reply:
> Host OS -- should we use "Node OS"

Fixed in [8eeb6e9](https://github.com/cntt-n/CNTT/pull/3217/commits/8eeb6e9c166cc3a4166238034dc17583f2e5fbac)